app/lib/core/services/database_provider/db_services.dart

```diff
@@ -0,0 +1,212 @@
+import 'dart:async';
+import 'package:connectivity/connectivity.dart';
+import 'package:mysql1/mysql1.dart';
+import 'package:app/core/models/user.dart';
+import 'package:app/core/models/animal_transport_record.dart';
+
+class DatabaseServices {
+  var networkStatus;
+  MySqlConnection dbConnect;
+
+  Future<bool> checkConnectivity() async {
+    var connectivityResult = await (Connectivity().checkConnectivity());
```

Comment:
- [ ] If using `await`, be sure to wrap it in a `try-catch` or some other Future-istic code, especially in the database (we don't do this with tests because they're supposed to work)